grakn-test/src/test/java/ai/grakn/test/graql/analytics/DebugAnalyticsTestIT.java

```diff
@@ -0,0 +1,19 @@
+package ai.grakn.test.graql.analytics;
+
+import ai.grakn.Grakn;
+import ai.grakn.GraknGraph;
+import ai.grakn.engine.util.ConfigProperties;
+import ai.grakn.test.AbstractScalingTest;
+import org.junit.Test;
+
+/**
+ *
+ */
+public class DebugAnalyticsTestIT extends AbstractScalingTest {
+
+    @Test
+    public void testSlowMethod() {
+        GraknGraph graph = Grakn.factory(Grakn.DEFAULT_URI, ConfigProperties.SYSTEM_GRAPH_NAME).getGraph();
+        System.out.println(graph.graql().compute().count().execute());
+    }
+}
```

Comment:
I have filed this as a bug for Filipe, he knows to look at my branch for the test so get rid of it